src/viewer.js

```diff
@@ -1217,6 +1218,10 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
 
         this._hideMessage();
 
+        $.extend ( true, options, {
```

Comment:
Doing this means the overall setting would override the setting passed directly into the function. It should be the other way around. You could swap `options` and the object literal and assign the result back to options, or go with something like: 

```
if (options.placeholderFillStyle === undefined) {
  options.placeholderFillStyle = _this.placeholderFillStyle;
}
```